core/src/main/java/io/varhttp/ControllerExecution.java

```diff
@@ -45,7 +47,16 @@ public void execute(ControllerContext context) {
 			filters.add(filterExecution);
 			Iterator<Object> iterator = filters.iterator();
 			VarServletFilterChain chain = new VarServletFilterChain(context, iterator.next(), iterator);
+			if (context.response().getHeader("Content-Type") == null) {
```

Comment:
shouldn't we always respect accepts headers? event if content-type is set

Reply:
yes